src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -411,13 +411,13 @@ public void testAdjustFeedbackResponseForEnrollments() throws Exception {
                                             Integer.parseInt(feedbackResponse1InBundle.feedbackQuestionId));
         FeedbackResponseAttributes responseBefore =
                 frLogic.getFeedbackResponse(feedbackQuestionInDb.getId(),
-                                            feedbackResponse1InBundle.giverEmail,
-                                            feedbackResponse1InBundle.recipientEmail);
+                                             feedbackResponse1InBundle.giver,
+                                             feedbackResponse1InBundle.recipient);
```

Comment:
These lines seem to have been shifted to the right by 1 space, probably unintended.